src/pages/chat/screenChat/screenChat.ts

```diff
@@ -73,11 +50,34 @@ export class ScreenChat extends Block {
           }
         }
       }
-    };
-    super(info);
+    });
+    document.addEventListener('keydown', (e)=>{
+      if (e.key === 'Enter' && !!this.currentMessage) {
+        this.methods.handlerSendMessage();
+      }
+    });
+    chatStore.on(StoreEvents.Updated, () => {
+      let currentIdUser = userStore.getState().id;
+      if (currentIdUser) {
+        this.currentIdUser = currentIdUser;
+      }
+    });
+
+    chatStore.on(StoreEvents.Updated, () => {
+      const messages = chatStore.getState().chatsMessages[this.props.idChat];
+
+      if (messages && messages.length) {
```

Comment:
Такие моменты удобно сокращать
```suggestion
      if (messages?.length) {
```